run_all_experiments.py

```diff
@@ -510,13 +511,15 @@ def _process_total_coverage_gain() -> dict[str, dict[str, Any]]:
       cov_relative_gain = 0.0
 
     total_lines = max(total_cov.total_lines, total_existing_lines)
+    union_linked_lines = max(total_cov.total_lines, total_existing_lines)
 
     if total_lines:
       coverage_gain[project] = {
           'language':
               oss_fuzz_checkout.get_project_language(project),
           'coverage_diff':
-              total_cov.covered_lines / total_lines,
+              coverage_utils.calculate_coverage_improvement(
+                  total_cov, existing_textcov, union_linked_lines),
```

Comment:
nit: I think we have subtracted `existing_textcov` above:
https://github.com/google/oss-fuzz-gen/blob/546036d8985be63e3b36017d2077742891e56655/run_all_experiments.py#L504-L506

While this action should be idempotent, repeated actions add unnecessary complexity and may confuse readers.